cli.hpp

```diff
@@ -441,6 +441,12 @@ struct SimulationConfig {
   }};
   uint64_t gml_bitrate = 0;
   uint32_t random_seed = 0;
+  uint64_t max_bitrate = 100 * 1'000'000;
+  Args::FlagInt<uint64_t> flag_max_bitrate{{
+      {"--max-bitrate"},
+      max_bitrate,
+      "Maximum bitrate per node",
+  }};
```

Comment:
For simplicity maybe pass bitrate in Mbps? So that we can set it by `--max-bitrate 100` to set it to 100Mbps